### ISSUE TITLE
Add a Raku blog site as "blogs.raku.org"

### ISSUE THREAD

**Issue by tbrowder** [Raku/problem-solving#231]
Except for the Advent Wordpress site, Raku does not have its own blogging platform.
There are many sites that may be and are used, but having our own easy-to-use site might
encourage more Raku blogging activity as well as visibility.

Some desirable traits:

+ minimal oversight 
+ input in Raku pod by PR
+ static website
+ authors apply for a user name
+ article entries by date and author (add tags as desired)
+ default page list articles by most recent
+ search by author or date or tag
+ articles to be correctible by PR

**Comment by lizmat** [comment]
This is not a production issue, even though it's related to infrastructure.  So I feel it would be an unnecessary burden for @rba : it's a community question that needs answering.  Once the answer is there, I'm pretty sure @rba will do whatever they can to make it happen :-)

**Comment by rba** [comment]
I agree. This has first to be discussed. As soon as the direction is clear it would be great to involve me again. Cheers.

**Comment by tbrowder** [comment]
Did I miss a "community" label?

**Comment by JJ** [comment]
Easiest thing is to create an Hugo or similar based one in GitHub. It can
be hosted at raku.github.io/blogs, anyone with permission can write, an so
on.
Another option would be to add another domain to raku-advent.blog, or
simply create a new rakublogs.wordpress.com site. We can simply add people
there...

But I wouldn't do it before we really, really, feel the need for it. So far
people are writing in their own blogs, and @lizmat spreads the word in the
raku weekly. Continuing that way is probably a good thing, at least until
the point it does not scale (meaning: too much work for Liz)

**Comment by CIAvash** [comment]
I don't know how much it's related, but #158 exists, which can support multiple blogs.

**Comment by tbrowder** [comment]
From my comments on #raku, I need a decent place to blog. (Decent means easy and NOT Wordpress.) A shared site is good and the support work can be shared, too.

**Comment by JJ** [comment]
El lun., 7 sept. 2020 a las 14:13, Tom Browder (<notifications@github.com>)
escribió:

> From my comments on #raku, I need a decent place to blog. (Decent means
> easy and NOT Wordpress.) A shared site is good and the support work can be
> shared, too.
>

OK, then. What about the version using Hugo? Would that be better?

**Comment by tbrowder** [comment]
I think so, at least as a start. And use a "real" domain name of something like the suggested "blogs.raku.org".

**Comment by coke** [comment]
-1. Let’s learn the lessons of blogs.perl.org. I think it makes sense to aggregate multiple blogs but not host them

**Comment by codesections** [comment]
> -1. Let’s learn the lessons of blogs.perl.org.

Here's a possibly ignorant question: just what _are_ the lessons of blogs.perl.org?  I've heard a few references to it as a negative lesson we should avoid, but I haven't been able to find anything that describes anything bad happening to it.  When I go there, it looks … basically decent, I guess?  But I never followed it (since I'm not/wasn't a Perl programmer).  Did it go through a patch of being abandoned?  Or was there interpersonal conflict?  Or a security issue?  I'm not asking for a detailed history lesson or anything, but I'd appreciate a link to more info or something – as is, I'm not quite sure what the lessons are

**Comment by coke** [comment]
This isn't intended to be an exhaustive list, but:

* dealing with spam
* managing accounts
* managing infrastructure
* managing the codebase, bugfixes..
* encouraging language blogging in a single location, rather than on multiple sites - isolationist.

Basically, there's a lot of work to support it, it would seem to keep us off in a corner, and honestly, it's pretty easy to find a place to put a blog together these days.
